js/column.js

```diff
@@ -2164,6 +2164,12 @@ Object.defineProperty(ForeignKeyPseudoColumn.prototype, "display", {
                 if (module._foreignKeyInputModes.indexOf(displ.selector_ux_mode) !== -1) {
                     sourceDisplay.inputDisplayMode = displ.selector_ux_mode;
                 }
+
+                // check visible-columns annotation display property and set to override property from fk annotation or display annotation
+                // make sure `false` is not ignored since that turns this feature off
+                if (displ.bulk_create_foreign_key !== null) {
```

Comment:
`null` is also a valid value here, right? So if I have

```
"vis-col": {
  "*": [
    { 
       "source": ...,
       "bulk_create_foreign_key": null
    }
  ]
}
```

and then on the fk
```
"foreign-key": {
  "display": {
    "*" : {
       "bulk_create_foreign_key": false
    }
  }
}
```

We're supposed to use he heuristics, right?